Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -387,6 +387,7 @@
         context.scale({ 1, -1 });
 
         [page drawWithBox:kPDFDisplayBoxCropBox toContext:context.platformContext()];
+        paintPDFOverlays(context);
```

Comment:
do you mean to do this for each page? or should this be two lines lower?

Reply:
No I did not, moved!